.eslintrc

```diff
@@ -30,9 +33,9 @@
       }
     },
     {
-      "files": ["./src/cli.ts"],
+      "files": ["./src/oclif/CustomCommand.ts"],
       "rules": {
-        "import/first": "off"
+        "@typescript-eslint/no-var-requires": "warn"
```

Comment:
Is there a `require` there? I think maybe this is not necessary

Reply:
Yes, it's here: ```const { config } = require('@oclif/errors')```

Reply:
Ohhh, I didn't saw it there